diff --git a/pyproject.toml b/pyproject.toml
index fe27c74..3e5837b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -53,6 +53,10 @@ cmake.version = 'CMakeLists.txt'
 ninja.version = '>=1.11'
 wheel.packages = []

+[[tool.scikit-build.overrides]]
+if.platform-system = 'win32'
+cmake.args = ['-G', 'Ninja', '-DCMAKE_POLICY_DEFAULT_CMP0141=OLD']
+
 [tool.scikit-build.metadata.version]
 provider = 'crysfml_version'
 provider-path = 'tools'
@@ -74,9 +78,6 @@ archs = ['AMD64']
 before-build = 'pip install delvewheel'
 repair-wheel-command = 'delvewheel repair -w {dest_dir} {wheel}'

-[tool.cibuildwheel.windows.environment]
-CMAKE_GENERATOR = 'Ninja'
-
 [tool.versioningit.format]
 distance = '{base_version}+dev{distance}'
 dirty = '{base_version}+dirty{distance}'
